argutils/viz.py

```diff
@@ -144,3 +155,18 @@ def nx_get_dot_pos(G):
         if n.get_pos() and n.get_name().isdigit()
     }
 
+def get_edge_spans(G, ts, max_edge_width):
+    """
+    Returns a list of length "num_edges" in G, the graphical representation of the
+    tree sequence, where each entry is the genomic span of the corresponding edge in the
+    tree sequence.
+    """
+    edge_spans = (ts.tables.edges.right - ts.tables.edges.left)
```

Comment:
This isn't quite right as we're thinking about graph theoretic edges here (u, v) rather than tskit ones (left, right, u, v).

The nx graph should already have an attribute ``intervals`` which has all (left, right) pairs associated with it. See the convert_nx (or whatever it's called) for details. You should be able to easily compute from that.

Reply:
There's a discussion about whether it should be a multigraph for viz purposes, but let's not worry about that for now.